### ISSUE TITLE
[QoS] qos.yml does not exist in sonic-mgmt

### ISSUE THREAD

**Issue by okanchou9** [Azure/sonic-mgmt#568]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

There is a QoS test in testcases.yml:

qos:
      filename: qos.yml
      topologies: [ptf32, ptf64]

Try to run the QoS test using the following CLI in my testbed:

ansible-playbook test_sonic.yml -i inventory --limit switch2 --become --extra-vars "testbed_name=ptf1-m testcase_name=qos" -vvvvvv

But got error log like following:

Tuesday 10 April 2018  10:22:07 +0800 (0:00:00.101)       0:00:13.174 ********* 
fatal: [switch2]: FAILED! => {"failed": true, "reason": "ERROR! the file_name '/home/testbed/SONiC/Azure/ansible/qos.yml' does not exist, or is not readable"}

Also I cannot find any yml file named "qos.yml" in folder role/test/task, does anyone face this issue before?

**Comment by lvphuc133** [comment]
Yes, I have the same problem. There is a "qos" folder in ./roles/test/task/ folder but there is nothing except a lua script there.

**Comment by crzas** [comment]
I have the same problem, too. it seems the file is missed!!!
